<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="1" time="1.260" timestamp="2024-09-24T11:51:31.824274+00:00" hostname="ip-172-31-30-46"><testcase classname="" name="tests.unit.test_app" time="0.000"><error message="collection failure">venv/lib/python3.9/site-packages/_pytest/python.py:493: in importtestmodule
    mod = import_path(
venv/lib/python3.9/site-packages/_pytest/pathlib.py:582: in import_path
    importlib.import_module(module_name)
/usr/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
&lt;frozen importlib._bootstrap&gt;:1030: in _gcd_import
    ???
&lt;frozen importlib._bootstrap&gt;:1007: in _find_and_load
    ???
&lt;frozen importlib._bootstrap&gt;:986: in _find_and_load_unlocked
    ???
&lt;frozen importlib._bootstrap&gt;:680: in _load_unlocked
    ???
venv/lib/python3.9/site-packages/_pytest/assertion/rewrite.py:175: in exec_module
    source_stat, co = _rewrite_test(fn, self.config)
venv/lib/python3.9/site-packages/_pytest/assertion/rewrite.py:355: in _rewrite_test
    tree = ast.parse(source, filename=strfn)
/usr/lib/python3.9/ast.py:50: in parse
    return compile(source, filename, mode, flags,
E     File "/var/lib/jenkins/workspace/microblog_prac_main/tests/unit/test_app.py", line 3
E       subprocess.run([/tests/unit/test_app.sh])
E                       ^
E   SyntaxError: invalid syntax</error></testcase></testsuite></testsuites>